source/windows/bcrypt_aes.c

```diff
@@ -555,120 +536,162 @@ struct aws_symmetric_cipher *aws_aes_cbc_256_new_impl(
     return NULL;
 }
 
-/* the buffer management for this mode is a good deal easier because we don't care about padding.
-   We do care about keeping the final buffer less than a block size til the finalize call so we can
-   turn the auth chaining flag off and compute the GMAC correctly. */
+/*
+ * The buffer management for this mode is a good deal easier because we don't care about padding.
+ * In chained mode, BCrypt expects the data to be passed in in multiples of block size,
+ * followed by a finalize call that turns off chaining and provides any remaining data.
+ * This function takes care of managing this state - you give it data to work and cipher state and
+ * it will return what data can be sent to bcrypt now and as side effect will update the cipher state
+ * with any leftover data.
+ * Note: this function takes a scratch buffer that might be used for to back data returned by the cursor.
+ * It is on caller to cleanup that scratch buffer.
+ */
+static struct aws_byte_cursor s_gcm_get_working_slice(
+    struct aes_bcrypt_cipher *cipher_impl,
+    struct aws_byte_cursor data,
+    struct aws_byte_buf *scratch) {
+    AWS_PRECONDITION(cipher_impl);
+    AWS_PRECONDITION(working_buf);
```

Comment:
wtf, I don't see where `working_buf` is declared ??

Reply:
hmm, i though we ran debug builds on win. this got lost after renaming working_buf to scratch. 

let me see whats going on with debug

Reply:
win debug ci was missing. added it in latest rev.